Snowflake.Data/Core/Session/SFSession.cs

```diff
@@ -129,6 +129,11 @@ internal Uri BuildLoginUrl()
             return loginUrl;
         }
 
+        private string EscapeDbObjectForUrl(string name)
+        {
+            return name == string.Empty ? name : $"\"{name.Replace("\"", "\"\"")}\"";

```

Comment:
Please make sure the name isn't already wrapped with quotes.
Make sure if connection tests are passing after this fix.